build/BuildDefaults.props

```diff
@@ -28,7 +28,7 @@
 
     <!-- Disable asset target fallback warning globally since it does not work transitively on NoWarn of individual packages -->
     <!-- Since we disabled the implict fallback to net461, this will only kick in when we have an explicit fallback and we don't need to be warned about it doing what we asked it to do. -->
-    <NoWarn>NU1701</NoWarn>
+    <NoWarn>NU1701;NU5104</NoWarn>
```

Comment:
I think it's worthwhile to update the comment to say it's describing `NU1701`, and add a description of the `NU5104`, since we don't know how long we'll be ignoring this warning.

Reply:
Done...